packages/commonwealth/client/styles/components/sidebar/sidebar_quick_switcher.scss

```diff
@@ -1,55 +1,41 @@
 @import 'shared';
 
 .SidebarQuickSwitcher {
-  border-right: 1px solid $neutral-50;
+  background: $neutral-50;
+  border-right: 1px solid $neutral-200;
   display: flex;
   flex-direction: column;
+  align-items: center;
   height: 100%;
   overflow: hidden;
-  min-width: $quick-switcher-width;
+  width: $quick-switcher-width;
+
+  &.chain-scoped {
+    padding: calc(#{$sidebar-community-header-height} + 4px) 4px 4px;
```

Comment:
Why do it this way instead of just having the presence of `CommunityHeader` push the others down?